packages/@ama-sdk/core/cli/update-spec-from-npm.cts

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
```

Comment:
would be maybe simpler to generate this file in `.cjs` within the sdk repo

Reply:
I don't see how this would make things simpler, at least from the user point of view

Reply:
For the user point of view it will have access to the script that it can trick if needed. It will also reduce the number of required inputs.

For our point of view we do not need to expose a new CLI.

Reply:
I personally see this as unnecessary boilerplate in the generated SDK, and more difficult to update with migration scripts

Reply:
(I am still not convince but) Ok then you will need to provide documentation (and probably UTs) to the CLI because SDK user will not have access to it directly.

Reply:
May need to be referred in the readme.md as well :)